modules/samples/producers/wso2-event-performance/src/main/java/org/wso2/carbon/sample/performance/Client.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.sample.performance;
+
+import org.apache.log4j.Logger;
+import org.wso2.carbon.databridge.agent.AgentHolder;
+import org.wso2.carbon.databridge.agent.DataPublisher;
+import org.wso2.carbon.databridge.commons.Event;
+
+import java.text.DecimalFormat;
+import java.util.Arrays;
+import java.util.Random;
+
+public class Client {
+    private static Logger log = Logger.getLogger(Client.class);
+
```

Comment:
Hi Lansatha,

Thank you very much sharing valuable content regarding warming up the servers. I have enabled warming up by letting user to specify warmUpCount and those events will not be logged. 

On the other hand regarding random generation, we thought of doing a test by generating events with values by random generator and compare durations with generating events with hard coded values. If there is a performance issue, we will pool the random numbers and re use them instead of re generating them.

Again regarding nanoTime vs millisecond i found following comparisons.  
http://www.javamex.com/faq/timing.shtml
https://blogs.oracle.com/dholmes/entry/inside_the_hotspot_vm_clocks